.kokoro/build-with-run.sh

```diff
@@ -32,6 +32,45 @@ popd
 export GOOGLE_CLOUD_PROJECT=nodejs-docs-samples-tests
 pushd github/nodejs-docs-samples/${PROJECT}
 
+# Verify changes worth testing.
+ignore_pattern='\.md$|^\.github|\.gitignore|^LICENSE|^CODEOWNERS|^\.eslint|\.prettier|^linkinator|^renovate'
```

Comment:
**Nit:** I initially thought this referred to **commit names** due to the `renovate` keyword! 😬 

```suggestion
ignored_files='\.md$|^\.github|\.gitignore|^LICENSE|^CODEOWNERS|^\.eslint|\.prettier|^linkinator|^renovate'
```